src/Command/GenerateObjectAclCommand.php

```diff
@@ -54,7 +54,7 @@ class GenerateObjectAclCommand extends QuestionableCommand
      */
     private $registry;
 
-    public function __construct(Pool $pool, array $aclObjectManipulators, RegistryInterface $registry = null)
+    public function __construct(Pool $pool, array $aclObjectManipulators, $registry = null)
```

Comment:
Please add type hints here

Reply:
As I see it the problem is that there is no base class or interface that both implement and that I can be sure is available from Symfony version 2.8 until 4.4.

Reply:
This is the interface that would be the replacement:
https://github.com/doctrine/persistence/blob/master/lib/Doctrine/Persistence/ManagerRegistry.php
But this exists not in any version 1 of this package. :(

Reply:
I know, that there are two different base classes, so please add a PHPDoc for this and an internal assert for one of those classes